dz26/src/pages/CategoryList/CategoryList.module.css

```
@@ -0,0 +1,44 @@
+.ul {
+  display: grid;
+  grid-template-columns: repeat(3, auto);
+  row-gap: 40px;
+  justify-content: space-between;
+  width: 100%;
+  max-width: 800px;
+  margin: 0 auto;
+  margin-top: 30px;
+}
+
+.li {
+  color: #fdf7fd;
+  cursor: pointer;
+
+  &:nth-child(3n + 1) {
+    background-image: linear-gradient(45deg, #1f1513 0% 50%, #6e2920 50% 100%);
+    border-bottom: 5px solid #6e2920;
+  }
+
+  &:nth-child(3n+2) {
+    background-image: linear-gradient(45deg, #0e1512 0% 50%, #1b5745 50% 100%);
+    border-bottom: 5px solid #1b5745;
+  }
+
+  &:nth-child(3n + 3) {
+    background-image: linear-gradient(45deg, #0d1520 0% 50%, #004074 50% 100%);
+    border-bottom: 5px solid #004074;
+  }
+
+  &:last-child {
+    color: #c2f0c2;
+  }
+}
+
+.button {
+  position: relative;
+  right: 50px;
```

```suggestion
  margin-right: 50px;
```

можно просто использовать отступ справа